src/main/java/ro/secur/auth/configuration/SecurityConfiguration.java

```diff
@@ -50,7 +50,8 @@ protected void configure(HttpSecurity http) throws Exception {
                 .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
                 .and()
                 .authorizeRequests()
-                .antMatchers(RESET_PASSWORD_URL, FORGOT_PASSWORD_URL, USERS_URL).permitAll()
+                .antMatchers(USERS_URL).access("hasAuthority('ADMIN')")
```

Comment:
You can call directly hasAuthority without calling access method. Besides that please also use the enum values from the Role to avoid hardcoded strings.